charts/rancher-gatekeeper-operator/v3.1.0-beta.700/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: A Helm chart for Gatekeeper
+name: rancher-gatekeeper-operator
+keywords:
+  - open policy agent
+version: v3.1.0-beta.700
```

Comment:
Based on the strict helm semver comparisons, I wonder if we should start with v0.1.0 for this helm chart until it actually GA's and then use the official version from Gatekeeper.

Reply:
not sure about the reason, will discuss this